### ISSUE TITLE
Support to choose database type: Postgres or  MariaDB

### ISSUE THREAD

**Issue by 3atharvkurdukar** [rtCamp/Frappe-Manager#229]
**Description**
The Frappe framework supports both MariaDB and Postgres as databases. Please add support in frappe manager as well.

**Use Case**
This will allow users to create Postgres-based apps using frappe manager

**Expected Behavior**
`--db-type` option should be available for `fm create` command, which will setup the Postges DB instead of MariaDB.

**Current Behavior (if applicable)**
Only MariaDB is available.

**Comment by kumarraditya** [comment]
Does Frappe framework support postgres db completely , am not much bother about other applications ?

**Comment by niraj2477** [comment]
>Does Frappe framework support postgres db completely

No, ERPNext does not work properly with postgres. IMO it is better to use MariaDB only
